subgraph.template.yaml

```diff
@@ -34,6 +34,35 @@ dataSources:
         - event: LogFinalize(indexed bytes32,indexed bytes32)
           handler: handleFinalize
       file: ./src/RealitioMapping.ts
+  - kind: ethereum/contract
+    name: GelatoCore
+    network: {{network}}
+    source:
+      address: '{{GelatoCore.address}}'
+      abi: GelatoCore
+      startBlock: {{GelatoCore.startBlock}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.3
+      language: wasm/assemblyscript
+      entities:
```

Comment:
Entities are the `@entity` definitions in the schema.graphql, not events or handlers.

Reply:
fixed.